src/events/event-stream.go

```diff
@@ -0,0 +1,22 @@
+package events
```

Comment:
I think we could group more of the logic together and do something prettier but considering we only have one route for now it's fine. Seems good to me